package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-migrations",
-  "version": "1.0.4",
+  "version": "1.1.0",
```

Comment:
Afaik this is something sematic-release takes care of. We don't need to handle release-related things on PRs.

Reply:
Once the semantic-release is merged 🙏